### PR TITLE
Initial pyjade port to pypugjs

### PR DESCRIPTION
**Pull request by dol-sen**
Signed-off-by: Brian Dolbec <dolsen@gentoo.org>

This just swaps out the use of pyjade code for a maintained fork of the original pyjade code.
I have a paired down version of this patch which applies to the 2.7.0 release for our Gentoo package repository.  I have tested buildbot both with the test suite and running the example buildbot "Hello world"  master.cfg

### CODE REVIEW

File: master/docs/manual/configuration/www.rst [hunk 1]
```diff
@@ -131,6 +131,8 @@ This server is configured with the ``www`` configuration key, which specifies a
         * quotes in attributes are not quoted. https://github.com/syrusakbary/pyjade/issues/132
```
Comment (tardyp) [inline master/docs/manual/configuration/www.rst:131]: we should probably make a quick test and see if this is still the case.

**Comment by tardyp** [comment]
Looks good. If we can verify that pyjade/132 is fixed that would actually make a nice improvement. We will need a news fragment as the upgrade will not be seemless as an optional dependency is changing.

File: master/docs/manual/configuration/www.rst (hunk 1 above)
Comment (dol-sen) [reply master/docs/manual/configuration/www.rst:131]: OK, the new code is identical aside from some minor reformatting. So, no it has not been fixed.

The new code is at lines 145 to 155 of nodes.py

**Comment by dol-sen** [comment]
I've opened this issue with pypugjs:
Double quotes in attribute (carry-over bug from pyjade issue #132) #60
at https://github.com/kakulukia/pypugjs/issues/60

Hopefully it can be fixed quickly.  Have you got an up-to-date link to a file that this fails for?  I can take a look at fixing the code to submit a PR.

**Comment by p12tic** [comment]
@dol-sen Could you add a newsfragment? I would then merge the PR. Thanks a lot!

**Comment by dol-sen** [comment]
Did tardyp get that bug fixed?  We did confirm the bug still exists in pyjade.  He was working on a fix.

**Pull request merged**